tsfile/src/main/java/org/apache/iotdb/tsfile/compress/Compressor.java

```diff
@@ -27,9 +23,9 @@
 /**
  * compress data according to type in schema.
  */
-public abstract class Compressor {
+public interface Compressor {
```

Comment:
Maybe the name `Compressor` should also be changed in corresponding to interface, like `ICompressor`.

Reply:
Modified.